packages/react-native/types/bugsnag.d.ts

```diff
@@ -1,13 +1,12 @@
-import { Client, BugsnagStatic, Config } from '@bugsnag/core'
+import { Client, Config } from '@bugsnag/core'
 
 // these properties are allowed to be configured in the JS layer
 type Configurable = 'onError' | 'onBreadcrumb' | 'logger' | 'metadata' | 'user' | 'context' | 'plugins'
 
 type ReactNativeConfig = Pick<Config, Configurable>
 
-interface ReactNativeBugsnagStatic extends BugsnagStatic {
-  start(apiKeyOrOpts?: string | ReactNativeConfig | LoadedConfig): Client
-  createClient(apiKeyOrOpts?: string | ReactNativeConfig | LoadedConfig): Client
```

Comment:
Are multiple clients no support in RN?

Reply:
No – it doesn't make sense as it is backed by a singleton native notifier so everything is would be routed by that anyway.